feature/space/src/main/java/com/yapp/itemfinder/space/itemdetail/ItemDetailSideEffect.kt

```diff
@@ -0,0 +1,7 @@
+package com.yapp.itemfinder.space.itemdetail
+
+import com.yapp.itemfinder.feature.common.SideEffect
+
+sealed class ItemDetailSideEffect : SideEffect {
+    object MoveToEditItem : ItemDetailSideEffect()
```

Comment:
`p2;` 추후 itemId값 넣어주세요~
```suggestion
    data class MoveToEditItem(val itemId: Long) : ItemDetailSideEffect()
```